src/fonts/ionicons.scss

```diff
@@ -8,6 +8,7 @@ $ionicons-font-path: $font-path !default;
 
 @import "ionicons-icons";
 @import "ionicons-variables";
+@import "~hail2u-scss-functions/string/_str-replace";
```

Comment:
Hmm.. ok
How about for starters, do:
`ion-icon[area-label^="arrow"] { @extends [flip] }` ?
To be honest, most of the icons don't matter, its just the arrows that are going the other way..